physlr/physlr.py

```diff
@@ -1425,6 +1464,8 @@ def assign_read_molecule(mxs, moltomxs, mol_counts, bx):
     @staticmethod
     def detect_communities_biconnected_components(g, node_set):
         """Separate bi-connected components. Return components."""
+        # if len(node_set) < Physlr.args.skip_small:
+        #     return [set(node_set)]
```

Comment:
Are we not skipping small sets now?

Reply:
Currently we skip small sets only when we are running `cos` and `sqcos`. I have added this code to all function as a reminder for later improvements that we need something ~similar (but not exactly the same as this) for other functions. I can also put a single-line reminder comment if we prefer it that way.